shared/helpers/puppeteer.js

```diff
@@ -146,15 +146,16 @@ function goToExample (page, exampleName) {
  * @param {import('puppeteer').Page} page - Puppeteer page object
  * @param {string} componentName - Component name
  * @param {object} [options] - Component options
- * @param {string} options.exampleName - Example name
+ * @param {string} [options.exampleName] - Example name
+ * @param {string} [options.queryParam] - Query param
  * @returns {Promise<import('puppeteer').Page>} Puppeteer page object
  */
 function goToComponent (page, componentName, options) {
   const componentPath = options?.exampleName
     ? `/components/${componentName}/${options.exampleName}/preview`
     : `/components/${componentName}/preview`
 
-  return goTo(page, componentPath)
+  return goTo(page, options?.queryParam ? `${componentName}?${options.queryParam}` : componentPath)
```

Comment:
The screenshots are currently screenshots of 404s, I think because this needs changing…

```suggestion
  return goTo(page, options?.queryParam ? `${componentPath}?${options.queryParam}` : componentPath)
```